integreat_cms/static/src/css/style.scss

```
@@ -561,7 +561,61 @@ datalist option {
 }
 
 /* Special word break for feedback list */
-.feedback-comment-full {
-    @apply break-words whitespace-pre-line;
-    word-break: break-word;
+.feedback-entry {
+    display: fleX;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: no-wrap;
```

```suggestion
        @apply overflow-hidden overflow-ellipsis whitespace-nowrap;
```
I'd use the tailwind classes here